### PR TITLE
Added Text Hierarchy in community/handbook/faq

### PR DESCRIPTION
**Pull request by Tabhi109**
**Description**

# This PR fixes 
- Added `h3` for questions
- Made `layer5` and `A:` as bold text.

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 632479082b65a333d515b4c89e9cd14349e9466c at: https://64dba3d11db2147d32ab6252--layer5.netlify.app

**Comment by HetviSoni** [comment]
Hi @Tabhi109 thank you for your contribution. Well, it is preferred that you raise an issue->get assigned->create pr, instead of direct pr.

File: src/sections/Community/Handbook/faq.js [hunk 1]
```diff
@@ -17,43 +17,43 @@ const Faq = () => {
...
 
-            <h4>Q: How do I join the mailing list?</h4>
-            <p>A: You can subscribe to the Layer5 mailing <a href="https://layer5.io/subscribe">here.</a></p>
+            <h3>Q: How do I join the mailing list?</h3>
+            <p><strong>A:</strong> You can subscribe to the <strong>Layer5</strong> mailing <a href="https://layer5.io/subscribe">here.</a></p>
 
-            <h4>Q: How do I get an internship in Layer5?</h4>
-            <p>A: To best position your candidacy for an internship with Layer5, engage in the community and its projects. Start contributing and keep contributing. Community members who consistently contribute are the first individuals to be awarded internships when new internship opportunities open up - particularly members who are making significantly impactful contributions.</p>
+            <h3>Q: How do I get an internship in <strong>Layer5</strong>?</h3>
```
Comment (HetviSoni) [inline src/sections/Community/Handbook/faq.js:32]: While you are on this page , you can change it to " How do I get an internship at Layer5?"

**Comment by abhijeetgauravm** [comment]
@Tabhi109 also do try to mention This PR fixes #issue_no .

File: src/sections/Community/Handbook/faq.js (hunk 1 above)
Comment (Tabhi109) [reply src/sections/Community/Handbook/faq.js:32]: Okay Sure , anything else you want me add or change ?

**Comment by Tabhi109** [comment]
@abhijeetgauravm , Okay thank you , It was my first time contributing , I'll surely keep that in mind from now on.

**Comment by l5io** [comment]
🚀 Preview for commit 731417a52421caffa577002b3352b01e6638d42e at: https://64de4148f3a3b421f8cb82b4--layer5.netlify.app

**Comment by vishalvivekm** [comment]
@Tabhi109 Let's discuss it on the websites call on Monday. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima).

**Comment by Tabhi109** [comment]
@vishalvivekm , I have added in the meeting minutes,Sure we will discuss it in the  website meeting.

**Review by Chadha93 — COMMENTED** [review]
@Tabhi109, this update isn't required. No need to <stong> Layer5.

**Comment by Chadha93** [comment]
> Hi @Tabhi109 thank you for your contribution. Well, it is preferred that you raise an issue->get assigned->create pr, instead of direct pr.

@Tabhi109 Let's follow this.

**Comment by Tabhi109** [comment]
Yes Sure !, I'll be careful next time , Thank you for telling me.

**Comment by Chadha93** [comment]
> Yes Sure !, I'll be careful next time , Thank you for telling me.

@Tabhi109 You can close this PR. Feel free to take open issues.

**Comment by Tabhi109** [comment]
Okay Sure !

**Pull request closed without merging**